Automattic/wp-calypso
# Stats: Append feature flag to purchase URL for WPCOM sites testing purchase page

Related to #

## Proposed Changes

* Append the feature flag `stats/paid-wpcom-stats` to stop redirecting WPCOM sites from visiting the Stats purchase page.

## Testing Instructions



* Spin up the change with the Calypso Live link.
* Navigate to Stats > `Traffic` page with the feature flag for a `Simple` site without any WPCOM plans or Stats products: `/stats/day/{site-slug}?flags=stats/paid-wpcom-stats`
* Click the `Upgrade my Stats` button of the upsell notice.
* Ensure the page redirects to the Stats purchase page.

## Pre-merge Checklist
